scripts/eslint/v7.js

```diff
@@ -0,0 +1,217 @@
+module.exports = {
+  extends: ['airbnb', 'prettier'],
+  parser: '@typescript-eslint/parser',
+  // // can be removed once we extend airbnb config
+  // parserOptions: {
+  //   ecmaVersion: 2018,
+  //   ecmaFeatures: { jsx: true },
+  //   sourceType: 'module',
+  // },
+  plugins: ['@typescript-eslint', 'deprecation', 'import', 'jest', 'jsx-a11y', 'react', 'react-hooks'],
+  // plugins: ['@typescript-eslint', 'deprecation', 'import', 'jest', 'jsx-a11y', 'react', 'react-hooks'],
+  // // can be removed once we extend airbnb config
```

Comment:
can remove already since we are extending airbnb?

Reply:
This and the next comment are some of the things I meant when I said this is a work in progress. :) Wanted to keep this section while I'm trying out a different approach, but it will be removed before check-in for sure.